src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/domain/timeline/Link.kt

```diff
@@ -0,0 +1,6 @@
+package uk.gov.justice.digital.hmpps.educationandworkplanapi.domain.timeline
+
+data class Link(
```

Comment:
I wasn't sure what this is initially (some kdoc would be good!), but after looking at the wireframe, I guess it represents the links to "View details" or "view assessment" etc?

I think I understand why you need the href here (since the event in question belongs in different domain and we might not know (or want to know) how to derive it?), but I think this needs some further thought/discussion/design.

Reply:
Added kdoc 👍